lib/react_on_rails/utils.rb

```diff
@@ -146,7 +146,7 @@ def required(arg_name)
     end
 
     def self.prepend_cd_node_modules_directory(cmd)
-      "cd #{ReactOnRails.configuration.node_modules_location} && #{cmd}"
+      "cd #{Regexp::escape(ReactOnRails.configuration.node_modules_location.to_s)} && #{cmd}"
```

Comment:
Why is this the right solution for shell escaping?

Reply:
What if put quotes around the value:

```ruby
"cd \"#{ReactOnRails.configuration.node_modules_location}\" && #{cmd}"
```

Reply:
Good idea! Didn't think of that. I like that solution better!